product-service/handlers/getProductsById.ts

```diff
@@ -0,0 +1,25 @@
+import {Context, APIGatewayEvent, APIGatewayProxyResult} from "aws-lambda";
+import {getProductsByIdWithDelay} from "service/products";
+
+export async function getProductsById(
+    event: APIGatewayEvent,
+    context: Context
+): Promise<APIGatewayProxyResult> {
+    try {
+        return {
+            statusCode: 200,
+            headers: {
+                "Access-Control-Allow-Origin": "*"
+            },
+            body: JSON.stringify(await getProductsByIdWithDelay(event.pathParameters.id, 10)),
+        };
+    } catch (e) {
+        return {
+            statusCode: 200,
```

Comment:
I think it will be better to return statusCode 404 if product is not found.

Reply:
I agree. Thanks. Fixed.